src/sentry/deletions/defaults/apigrant.py

```diff
@@ -11,3 +16,7 @@ class ModelApiGrantDeletionTask(ModelDeletionTask):
     def mark_deletion_in_progress(self, instance_list):
         # no status to track
         pass
+
+    def delete_instance(self, instance):
+        with unguarded_write(router.db_for_write(ApiGrant)):
```

Comment:
`ApiGrant` has a cascade into `SentryAppInstallation` via `SentryApp`.  It's ok not to generate the outbox in this case -- the cascading will result in ApiToken itself being deleted which will reap by other means.